packages/plugins/auto-doc/package.json

```
@@ -1,7 +1,7 @@
 {
 	"name": "@oscd-plugins/auto-doc",
 	"private": true,
-	"version": "1.14.0",
+	"version": "1.14.1",
```

I think this PR can be considered as a new feature to the table block, and as such be incremented as `1.15.0`

```suggestion
	"version": "1.15.0",
```